recce/__init__.py

```diff
@@ -55,4 +58,17 @@ def get_version():
         return version
 
 
+def fetch_latest_version():
+    try:
+        url = 'https://pypi.org/pypi/recce/json'
+        response = requests.get(url)
```

Comment:
I think we can add a timeout here. What do you think?
https://requests.readthedocs.io/en/stable/user/advanced/#timeouts

Reply:
Configure 3 3-second timeout. And if the version contains `dev`, it won't trigger fetch version mechanism.